### PR TITLE
 Add an opinionated article about Hacktoberfest 

### PR DESCRIPTION
**Pull request by cptcrunchy**
My personal thoughts about Hacktoberfest and contributing to open source projects

### CODE REVIEW

File: content/articles/2020_10_5_Hacktoberfest_Or_Spamtoberfest?.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+of our tech community and even grow as a developer. 
+
+Maybe your _first_ PR is editing a README, that's okay. Challenge yourself to make more "meaningful"
+contributions to the open source community and projects.
+
+
+
+
+
+
```
Comment (gtmanfred) [inline content/articles/2020_10_5_Hacktoberfest_Or_Spamtoberfest?.md:56]: Can you remove these extra newlines?

File: content/articles/2020_10_5_Hacktoberfest_Or_Spamtoberfest?.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+Tags: opinion, article
+Slug: hacktoberfest-or-spamtoberfest
+Authors: Jason Gutierrez
+Summary: An opinion on contributing to open source projects during Hacktoberfest
+
+# To Hacktober or Not to Hacktober?
+
+So in light of some recent events, mostly from a Youtuber with a large following,
+there was a "wave" of "spammy" PRs (Pull Requests) where "contributors" would
+add some text without out much context and that would be considered a valid PR.
```
Comment (gtmanfred) [inline content/articles/2020_10_5_Hacktoberfest_Or_Spamtoberfest?.md:12]: `out out`

File: content/articles/2020_10_5_Hacktoberfest_Or_Spamtoberfest?.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+I sympathize with the Youtuber and I believe they had good intentions but I believe
```
Comment (gtmanfred) [inline content/articles/2020_10_5_Hacktoberfest_Or_Spamtoberfest?.md:14]: `, but`

File: content/articles/2020_10_5_Hacktoberfest_Or_Spamtoberfest?.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+this lines up with the overall mentality of Imposter Syndrome which many of us, including
+myself, share. I think the concept of contributing to an open sourced project is
+noble and some projects are only alive because of a community that enjoys the service
```
Comment (gtmanfred) [inline content/articles/2020_10_5_Hacktoberfest_Or_Spamtoberfest?.md:17]: `, and`

File: content/articles/2020_10_5_Hacktoberfest_Or_Spamtoberfest?.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+or project but I think there is some underlying barrier that can scare off would be
```
Comment (gtmanfred) [inline content/articles/2020_10_5_Hacktoberfest_Or_Spamtoberfest?.md:18]: This has become a run on sentence. Can you reword it?

File: content/articles/2020_10_5_Hacktoberfest_Or_Spamtoberfest?.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+contributors.
+
+These contributors instead only "contribute" by correcting `README`s. Though in
+some cases this is necessary, from what I have read and experienced, this is not the case.
```
Comment (gtmanfred) [inline content/articles/2020_10_5_Hacktoberfest_Or_Spamtoberfest?.md:22]: Though, in some cases, this is necessary, from what I have read and experienced, that is not the case.

File: content/articles/2020_10_5_Hacktoberfest_Or_Spamtoberfest?.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+Maybe the results of the "spam" PRs is due to some unintentional intimidation that
+even some more seasoned developers have experienced.
+
+In light of these outcomes and sentiments I believe that the open source community
+shouldn't suffer because of this but rather "inform" would be contributors and follow
```
Comment (gtmanfred) [inline content/articles/2020_10_5_Hacktoberfest_Or_Spamtoberfest?.md:27]: Again, another run on sentence.

also, in the middle it is not super clear what "this" is.  Reworking the sentence would be good.

File: content/articles/2020_10_5_Hacktoberfest_Or_Spamtoberfest?.md [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+a sort of "honor" system. It should be considered a "given" or even an expectation that
+open source projects would be subject to being "spammed". Perhaps if these projects were more
+prepared and prefaced with a more "beginner" friendly set of instructions they could
```
Comment (gtmanfred) [inline content/articles/2020_10_5_Hacktoberfest_Or_Spamtoberfest?.md:30]: instructions, they could

File: content/articles/2020_10_5_Hacktoberfest_Or_Spamtoberfest?.md [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+reduce the "contributor" learning curve.
+
+## Recommendations to contributors
+
+If you are looking to contribute to an open source project be mindful of your contribution.
```
Comment (gtmanfred) [inline content/articles/2020_10_5_Hacktoberfest_Or_Spamtoberfest?.md:35]: project, be mindful

File: content/articles/2020_10_5_Hacktoberfest_Or_Spamtoberfest?.md [hunk 10]
```diff
@@ -0,0 +1,56 @@
...
+You may only interact with this project once in your life, more specifically only during Hacktoberfest.
+Imagine you are a "tourist" in France and you would like to add your "Lock of Love" to a
+particular chain-link fence. Your contribution may not seem like much but as a city employee
```
Comment (gtmanfred) [inline content/articles/2020_10_5_Hacktoberfest_Or_Spamtoberfest?.md:38]: like much, but as a city employee

File: content/articles/2020_10_5_Hacktoberfest_Or_Spamtoberfest?.md [hunk 11]
```diff
@@ -0,0 +1,56 @@
...
+that might have to "maintain" said fence, it can be overwhelming. 
+
+The city employee(s) that maintains the fence of locks is aware of these contributions 
+and should provide guidelines to would be "romeos" about the Do's and Don't of adding a lock to the fence.
+Order and community rules allow for a more enjoyable experience for all who chose to contribute.
+
+As a contributor, don't be afraid of rules or for that matter contributing. It is an experience that
```
Comment (gtmanfred) [inline content/articles/2020_10_5_Hacktoberfest_Or_Spamtoberfest?.md:45]: rules, or for that matter, contributing.

**Review by gtmanfred — CHANGES_REQUESTED** [review]
Thank you for writing this Jason.

I would love to get this up on the blog, I just have some grammar requests, mostly commas :)

**Comment by cptcrunchy** [comment]
I made the grammatical corrections. thanks for the review!

**Review by andrewthetechie — APPROVED** [review]
This LGTM, but let's wait to let @gtmanfred review again as well

**Review by gtmanfred — APPROVED** [review]

**Comment by gtmanfred** [comment]
looks good, thank you for the article @cptcrunchy

**Pull request merged**